app/views/interest/browse.scala.html

```diff
@@ -0,0 +1,30 @@
+@*****************************************************************************
+ * Page for browsing a subscriber's interests filtered by interest's scheme, *
+ * scheme's plan or other filter.                                            *
+ * Copyright (c) 2015 MIT Libraries                                          *
+ *****************************************************************************@
+@(id: Int, interests: List[Interest], filter: String, value: String, page: Int, total: Long)@*(implicit hubContext: HubContext)*@
+@layout.main("Interest Browse - TopicHub") {
+  <div class="page-header">
+    <h2>Browse Interests by @filter</h2>
+    @if("scheme" == filter && Subscriber.findById(id).get.plannedFor(Scheme.findByTag(value).get.id)) {
+      <h3>@value</h3>
+      <h3>Plan: @Subscriber.findById(id).get.planFor(Scheme.findByTag(value).get.id).get.name</h3>
+    }
+    @if("plan" == filter) {
```

Comment:
Yea, I can see that. But this is actually a recommended practice (and you see it a lot) in java code, or any code that works similarly (has nullable values), If you are comparing 2 strings and one could be null - call the constant C and the variable V - then:
if( V == C) or if (V.equals(C)) will throw a null pointer exception if V is null, *but*
if (C == V) of (C.equals(V)) will not, and will actually do what you you want....